license-tool/licenseTool.Dockerfile

```diff
@@ -16,6 +16,8 @@ RUN microdnf install -y git nodejs
 
 ARG MAVEN_VERSION=3.6.3
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+# https://github.com/eclipse/dash-licenses/commits Jul 21, 2020
+ARG DASH_LICENT_REV=b90756084cac437443b48f1d48edab6e991f2697
```

Comment:
So some reason even master started work as expected for me :smile: :gun: 
BTW it makes sense to define the version we rely on, and then configure some tool to upgrade the version if new is available after testing that it works for us.